SrfGen/gen_coords.py

```diff
@@ -7,8 +7,9 @@
 
 from qcore import qclogging
 from qcore.binary_version import get_unversioned_bin
-from qcore.constants import VM_PARAMS_FILE_NAME
-from qcore.utils import load_yaml
+from qcore.constants import VM_PARAMS_FILE_NAME, VMParams
```

Comment:
Is VM_PARAMS_FILE_NAME still needed?

Reply:
@shot0829  I agree - Fix this and I'll approve it.